src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerClientViewModel.cs

```diff
@@ -63,13 +63,14 @@ public TermsOfUseHelper(TermsOfUseHelperParams touParams)
             authenticationManager = touParams.AuthenticationManager;
         }
 
-        public void Execute()
+
+        public void Execute(bool preventReentrant = true)
```

Comment:
Break binary compatibility, though I don't think anyone else will use this class.